test/sql/include/plan_expand_hypertable_query.sql

```diff
@@ -146,6 +146,11 @@ SELECT * FROM cte ORDER BY value;
 :PREFIX SELECT * FROM hyper WHERE 10::bigint > time_bucket(10, time) ORDER BY time;
 :PREFIX SELECT * FROM hyper WHERE 11::bigint > time_bucket(10, time) ORDER BY time;
 
+-- test overflow behaviour of time_bucket exclusion
+SET client_min_messages TO error;
+:PREFIX SELECT * FROM hyper WHERE time > 950 AND time_bucket(10, time) < '9223372036854775807'::bigint ORDER BY time;
```

Comment:
is it possible to add one overflow case for timestamptz and interval?

Reply:
added more tests